app/build.gradle

```diff
@@ -138,9 +138,19 @@ android {
         // WebView
         ////////////////////////////////////////////////////////////////////////////////////////////
 
-        // Release
+        androidTest {
+            res.srcDir('src/androidTest/assets')
+            java.srcDirs += ['src/androidTest/java']
+        }
 
-        focusWebviewUniversalRelease {
+        androidTestWebviewCompile {
+            java.srcDirs += ['src/androidTestWebview/java']
+        }
+
+        androidTestGeckoCompile {
+            java.srcDirs += ['src/androidTestGecko/java']
+        }
+            focusWebviewUniversalRelease {
```

Comment:
Can you move those test configurations to the bottom of the block? I tried to create logical groups (see comments) for WebView/GeckoView and Release/Beta/Debug to that it's easier to get an overview.